crates/database/src/solver_competition.rs

```diff
@@ -57,14 +53,10 @@ pub async fn load_latest_competitions(
     latest_competitions_count: u32,
 ) -> Result<Vec<LoadCompetition>, sqlx::Error> {
     const QUERY: &str = r#"
-SELECT sc.json, sc.id, COALESCE(ARRAY_AGG(s.tx_hash) FILTER (WHERE so.block_number IS NOT NULL), '{}') AS tx_hashes
+SELECT sc.json, sc.id, COALESCE(ARRAY_AGG(s.tx_hash) FILTER (WHERE s.solution_uid IS NOT NULL), '{}') AS tx_hashes
```

Comment:
If we now use the existence/absence of this uid to detect solutions from the other environment it would be nice to add that to the DB readme info for that column.